scripts/create-index-pages.js

```diff
@@ -43,6 +43,9 @@ const getSubheading = (dir, level) => {
 };
 
 const createIndexPage = async (dir, level = 2) => {
+  // dont run for the attribute definitions
+  if (dir === path.join(BASE_DIR, 'attribute-dictionary')) return false;
```

Comment:
Also, ever since I read this article about the [Apple goto SSL bug](https://nakedsecurity.sophos.com/2014/02/24/anatomy-of-a-goto-fail-apples-ssl-bug-explained-plus-an-unofficial-patch/), I've exclusively used curlys to make sure I don't accidentally introduce bugs. I found that article to be a good reason to use them at all times.